scene/scene-layer-sample/src/main/java/com/esri/samples/scene/SceneLayerSample.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2015 Esri. Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law
+ * or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+package com.esri.samples.scene;
+
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.layers.ArcGISSceneLayer;
+import com.esri.arcgisruntime.mapping.ArcGISScene;
+import com.esri.arcgisruntime.mapping.ArcGISTiledElevationSource;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Surface;
+import com.esri.arcgisruntime.mapping.view.Camera;
+import com.esri.arcgisruntime.mapping.view.SceneView;
+
+public class SceneLayerSample extends Application {
+
+  private SceneView sceneView;
+  private static final String ELEVATION_IMAGE_SERVICE =
+      "http://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer";
```

Comment:
Some of these Scene Layers come with their own elevation service. I think it is best to use the one that you have here since it is the default one for all Scene Layers and it will be less confusing than using the individual one.

We could explain this in the ReadMe file about this one being the default and the one below being elevation just for this scene layer.
http://scene.arcgis.com/arcgis/rest/services/BREST_DTM_1M/ImageServer